### ISSUE TITLE
detect slug

### ISSUE THREAD

**Issue by dddesign** [slinkymanbyday/tracktry-card#1]
if I get it right we have to fill out name, tracking number and slug name. it would be easier to do it only by name and tracking number. the slug name could be found by tracking number, see https://www.tracktry.com/api-carriers-detect-carrier.html

**Comment by slinkymanbyday** [comment]
Yeah, the tracktry python package i made can call this api, the problem comes with what happens when it can't determine the slug? I will look into it sometime, but didn't have the time to go into it that much. Always happy to accept pull requests too

**Comment by dddesign** [comment]
it could act like on tracktry site: if you enter a tracking number it shows you possible slug names in the slug field.
(pull requests: I know some js but no python or typescript)
